### PR TITLE
Virtualized log for vx reports

### PR DESCRIPTION
**Pull request by normanrz**
Adds a virtualized list for rendering logs in vx reports. This is necessary because logs can get up to 100,000s entries long.

### Steps to test:
- Enable voxelytics and log streaming in application.conf (including setting up elasticsearch/opensearch locally)
- Stream some logs to wk
- Load the log tab in the workflow report, scroll around

------
- [x] Updated [(unreleased) changelog](../blob/master/CHANGELOG.unreleased.md#unreleased)
- [x] Ready for review

### CODE REVIEW

File: frontend/javascripts/admin/voxelytics/log_tab.tsx [hunk 1]
```diff
@@ -37,8 +42,108 @@ export function formatLog(
...
+
+  const charWidth = useMemo(() => {
+    const canvas = document.createElement("canvas");
+    const ctx = canvas.getContext("2d");
+    if (ctx == null) {
+      throw new Error("Could not create measuring canvas");
+    }
+    ctx.font = LOG_FONT;
+    const measurement = ctx.measureText("0123456789abcdefghijklmnopqrstuvwxyz");
+    console.log(measurement.width / 36);
```
Comment (philippotto) [inline frontend/javascripts/admin/voxelytics/log_tab.tsx:90]:
```suggestion
```

File: frontend/javascripts/admin/voxelytics/log_tab.tsx [hunk 2]
```diff
@@ -37,8 +42,108 @@ export function formatLog(
...
+  console.timeEnd("linecounting");
+
+  // eslint-disable-next-line react/no-unused-prop-types
+  function renderRow({ index, key, style }: { index: number; key: string; style: CSSProperties }) {
+    return (
+      <div
+        className={classnames("log-line", {
+          "log-line-even": index % 2 === 0,
+          "log-line-odd": index % 2 === 1,
+        })}
```
Comment (philippotto) [inline frontend/javascripts/admin/voxelytics/log_tab.tsx:121]:
```suggestion
        className={`log-line log-line-${index % 2 ? "odd" : "even"}`}
```

File: frontend/javascripts/admin/voxelytics/log_tab.tsx [hunk 3]
```diff
@@ -37,8 +42,108 @@ export function formatLog(
...
+    if (ctx == null) {
+      throw new Error("Could not create measuring canvas");
+    }
+    ctx.font = LOG_FONT;
+    const measurement = ctx.measureText("0123456789abcdefghijklmnopqrstuvwxyz");
+    console.log(measurement.width / 36);
+    return measurement.width / 36;
+  }, []);
+
+  console.time("linecounting");
```
Comment (philippotto) [inline frontend/javascripts/admin/voxelytics/log_tab.tsx:94]: can be removed, I assume.

**Comment by normanrz** [comment]
For the curious: Calculating the number of lines for a 193MB (9MB gzipped) json log file takes ~250ms on my laptop.

**Pull request merged**